scripts/rc.py

```diff
@@ -38,6 +38,16 @@ def add_ts_parsers(subparsers):
                         metavar="timesketch_name",
                         help="Delete the Base64 encoded timesketch name provided")
 
+    group.add_argument('--start',
+                        nargs='?',
+                        const="127.0.0.1",
+                        metavar="listener",
+                        help="Starts Timesketch Service")
+
+    group.add_argument('--stop',
```

Comment:
This is already in the allowed --service <start/restart/stop> timesketch function. So these are duplicative.  Or am I missing something?